src/code/z_bgcheck.c

```diff
@@ -4171,17 +4171,17 @@ u32 SurfaceType_IsWallDamage(CollisionContext* colCtx, CollisionPoly* poly, s32
 /**
  * Zora's Domain WaterBox in King Zora's Room
  */
-WaterBox zdWaterBox = { -348, 877, -1746, 553, 780, 0x2104 };
+WaterBox sZdWaterBox = { -348, 877, -1746, 553, 780, 0x2104 };
```

Comment:
I think this ought to be ZD if we must abbreviate it like this, since it's 2 words. But it's used so little I think it'd be fine to go the route of spelling it out in full.